rasa/train.py

```diff
@@ -431,10 +431,16 @@ def train_nlu(
 
     training_datas = await file_importer.get_nlu_data()
     if training_datas.is_empty():
-        print_error(
-            "No NLU data given. Please provide NLU data in order to train "
-            "a Rasa NLU model using the '--nlu' argument."
-        )
+        if nlu_data:
+            print_error(
+                f"Path '{nlu_data}' doesn't contain valid NLU data in it. "
```

Comment:
I think we should also explicitly state that the training of the NLU model will be skipped.